src/Matchers/Bruteforce.php

```diff
@@ -39,11 +39,12 @@ public function getFeedback($isSoleMatch)
         ];
     }
 
-    public function getRawGuesses()
+    public function getRawGuesses(): int
     {
         $guesses = pow(self::BRUTEFORCE_CARDINALITY, mb_strlen($this->token));
         if ($guesses === INF) {
-            return defined('PHP_FLOAT_MAX') ? PHP_FLOAT_MAX : 1e308;
+            // use 32bit max int if for some reason PHP_INT_MAX isn't defined
+            return defined('PHP_INT_MAX') ? PHP_INT_MAX : 2147483647;
         }
```

Comment:
Thanks for catching that!

Comparing against `PHP_INT_MAX` should work fine. Also just using PHP_INT_MAX since we no longer need to support versions of php which don't have that defined.